Cargo.toml

```diff
@@ -2,9 +2,9 @@ resolver = "2"
 
 [workspace]
 members = [
-    'node',
-    'runtime',
+    "node",
```

Comment:
nit: Single quotes were being used in all the TOML files since that kept the diff between `canvas` and `substrate-parachain-template` (our reference) small. 

For that reason I'd prefer to keep the single quotes, but would be open to changing them if you have a good reason as to why. Either way, this should be done in a standalone PR since it makes the diff pretty noisy